streampark-console/streampark-console-webapp/src/views/flink/app/components/AppView/StopApplicationModal.vue

```diff
@@ -79,6 +79,17 @@
         defaultValue: false,
         afterItem: () => h('span', { class: 'conf-switch' }, 'Send max watermark before stopped'),
       },
+      {
+        field: 'nativeFormat',
+        label: 'NativeFormat',
+        component: 'Switch',
+        componentProps: {
+          checkedChildren: 'ON',
+          unCheckedChildren: 'OFF',
+        },
+        defaultValue: false,
+        afterItem: () => h('span', { class: 'conf-switch' }, 'Use savepoint native format'),
+      },
```

Comment:
It's better hide this component when canceling without savepoint, besides, we may provide more info for users, like native format is supported since flink 1.15.